teuthology/task/ceph_ansible.py

```diff
@@ -21,11 +21,21 @@ class CephAnsible(Task):
 
     _default_playbook = [
         dict(
+            hosts='all',
+            become=True,
+            tasks=[],
+        ),
```

Comment:
the default playbook we are defining in the task has to run gatherfacts or else the master branch wont work, ideally we have use the site.yml.sample that comes with ceph-ansible instead of using the default playbook and it needs more testing across other branches, until that is done we should use this to unbreak master runs, more background at : https://github.com/ceph/ceph-ansible/issues/1498